aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIOProperties.java

```diff
@@ -225,6 +225,13 @@ public class S3FileIOProperties implements Serializable {
    */
   public static final String SESSION_TOKEN = "s3.session-token";
 
+  /**
+   * Configure the expiration time in millis of the static session token used to access S3FileIO.
+   * This expiration time is currently only used in {@link VendedCredentialsProvider} for refreshing
+   * vended credentials.
+   */
+  public static final String SESSION_TOKEN_EXPIRES_AT_MS = "s3.session-token-expires-at-ms";
```

Comment:
Can we just mark this as protected?  There's no reason to expose it since it would not be expected to be set externally (only through the vended path).

Reply:
done